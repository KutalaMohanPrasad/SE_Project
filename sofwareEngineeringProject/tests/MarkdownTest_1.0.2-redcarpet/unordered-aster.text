## Unordered

Asterisks tight:

*	asterisk 1
*	asterisk 2
*	asterisk 3


Asterisks loose:

*	asterisk 1

*	asterisk 2

*	asterisk 3
