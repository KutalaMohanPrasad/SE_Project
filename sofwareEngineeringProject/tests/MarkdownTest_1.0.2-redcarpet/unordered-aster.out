<h2>Unordered</h2>

<p>Asterisks tight:</p>

<ul>
<li>asterisk 1</li>
<li>asterisk 2</li>
<li>asterisk 3</li>
</ul>

<p>Asterisks loose:</p>

<ul>
<li><p>asterisk 1</p></li>
<li><p>asterisk 2</p></li>
<li><p>asterisk 3</p></li>
</ul>